.github/workflows/scripts/scopes-doc.py

```diff
@@ -9,7 +9,7 @@ def transformScopesToMarkDown(input):
 
     intro="""# Scopes für OAuth Clients bei Europace
 
-Mithilfe von Scopes kannst du genau angeben, welche Art von Zugriff du benötigst. Scopes beschränken den Zugriff für OAuth-Token. Sie gewähren keine zusätzlichen Berechtigungen über das hinaus, was der Benutzer bereits hat.
+Mit Hilfe der Scopes kannst du genau angeben, welche Art von Zugriff du benötigst. Scopes beschränken den Zugriff mittels OAuth-Tokens. Sie gewähren keine zusätzlichen Berechtigungen über die Rechte des Nutzers hinaus.
```

Comment:
```suggestion
Mithilfe der Scopes kannst du genau angeben, welche Art von Zugriff du benötigst. Scopes beschränken den Zugriff mittels OAuth-Token. Sie gewähren keine zusätzlichen Berechtigungen über die Rechte des Nutzers hinaus.
```

🤔  laut [Duden](https://www.duden.de/rechtschreibung/mithilfe_unterstuetzt_von) ist das die empfohlene Schreibweise. Im zweiten Satz ist das letzte `s` nicht zwingend. Der dritte Satz klingt wirklich besser 👍

Reply:
Oh je, arme deutsche Rechtschreibung. Aus "Mit Hilfe" wurde "mithilfe" ... Ich kenn nur die "Mithilfe" als Substantiv. 
"Die Mithilfe beim Umzug" oder "Mit Hilfe von X konnte ich den Umzug schaffen"

Reply:
Beim 's' ging es mir eher drum, dass technisch genau ein Access Token und nicht mehrere Token**s** erstellt werden.

Reply:
Das 's' kann gerne weggelassen werden.

Reply:
```suggestion
Mit Hilfe der Scopes kannst du genau angeben, welche Art von Zugriff du benötigst. Scopes beschränken den Zugriff mittels OAuth-Token. Sie gewähren keine zusätzlichen Berechtigungen über die Rechte des Nutzers hinaus.
```

Reply:
👍